app/viewmodels/CashAccountDailyStatementsViewModel.scala

```diff
@@ -42,15 +47,33 @@ case class CashAccountDailyStatementsViewModel(dailyStatements: Seq[DailyStateme
                                                noTransFromLastSixMonthsText: Option[HtmlFormat.Appendable] = None)
 
 object CashAccountDailyStatementsViewModel {
-  def apply(transactions: CashTransactions)(implicit msgs: Messages): CashAccountDailyStatementsViewModel = {
+  def apply(transactions: CashTransactions,
```

Comment:
we can probably keep the params together and then more the implicit down here?

Reply:
This is also fine and readable cause we try first to keep on the same line (if it does not go beyond 120 characters). But i have moved implicit to the next line